main.css

```diff
@@ -0,0 +1,282 @@
+a, a:link, a:active, a:visited, a:hover {
+	text-decoration: none;
+	outline: none;
+}
+ul {
+	list-style-type: none;
+}
+body {
+	color: #767e9e;
+	font-family: 'Lato', sans-serif;
+	font-size: 18px;
+	font-weight: normal;
+}
+.footer-information {
+	display: flex;
+	justify-content: space-between;
+}
+.inner {
+	margin: 0 auto;
+	max-width: 940px;
+	padding: 67px 10px;
+}
+.header {
+	display: flex;
+	justify-content: space-around;
+	align-items: center;
+	margin: 0 auto;
+	max-width: 940px;
+	padding: 34px 10px;
+}
+
+.header-main {
+	width: 100%;
+	background: #2d303a;
+	border-bottom: 6px solid #323746;
+}
+
+.icon {
+	background: url(/singolo/images/menu.svg);
+	background-size: cover;
+	width: 22px;
+	height: 22px;
+	display: none;
+}
+h1.logo {
+    width: 115px;
+    height: 20px;
+    background: url(/singolo/images/logo.png) no-repeat;
+}
+h1.logo a {
+    display: block;
+}
+
+ul.nav-menu {
+	padding: 9px 0 0 0;
+}
+ul.nav-menu li {
+	display: inline-flex;
+	flex-direction: row;
+	font-size: 12px;
+	font-weight: 700px;
+	color: #494e62;
+	text-transform: uppercase;
+}
+ul.nav-menu li a {
+	color: #ffffff;
+	margin: 0 13px 0 13px;
+}
+ul.nav-menu li a:hover {
+	color: #f06c64;
+	transition-duration: 0.5s;
+}
+ul.nav-menu li::after {
+      content: "\00B7";
+}
+.services {
+	background: #f2f2f2;
+	overflow: hidden;
+	border-bottom: 6px solid #ffffff;
+}	
+.services .inner {
+	padding-bottom: 31px;
+}
+.services ul {
+	padding: 46px 0 0 0;
+}
+.services ul li {
+	float: left;
+	padding: 0 20px 36px 0;
+	max-width: 290px;
+}
+.services ul li span {
+	float: left;
+	width: 56px;
+	height: 56px;
+	line-height: 56px;
+	text-align: center;
+	font-size: 35px;
+	font-family: 'linecons';
+	color: #666d89;
+	background: transparent;
+	border-radius: 50%;
+	border: 2px solid #666d89;
+	margin: 0 10px 0 0;
+}
+.services ul li:hover span {
+	border: 2px solid #f06c64;
+	transition-duration: 0.5s;
+}
+
+.li_heart:before {
+	content: url(/singolo/images/services-item-3--nocircle.svg);
+	vertical-align: middle;
+}
+.li_star:before {
+	content: url(/singolo/images/services-item-6--nocircle.svg);
+	vertical-align: middle;
+}
+.li_bulb:before {
+	content: url(/singolo/images/services-item-2--nocircle.svg);
+	vertical-align: middle;
+}
+.li_pen:before {
+	content: url(/singolo/images/services-item-1--nocircle.svg);
+	vertical-align: middle;
+}
+.li_bubble:before {
+	content: url(/singolo/images/services-item-5--nocircle.svg);
+	vertical-align: middle;
+}
+.li_phone:before {
+	content: url(/singolo/images/services-item-4--nocircle.svg);
+	vertical-align: middle;
```

Comment:
li_heart:before,
.li_star:before,
.li_bulb:before,
.li_pen:before,
.li_bubble:before,
.li_phone:before {
	vertical-align: middle;
}
What is it for? If you need to change vertical-align, you'll have to do it in one place, not in 6